src/components/switch/switch.scss

```diff
@@ -68,12 +68,6 @@ md-switch {
       right: -8px;
```

Comment:
We need to remove `:not([disabled])` from this rule as well so that the circle has a proper size when disabled switches are focused.